### ISSUE TITLE
Question: Is there a way I can add bussines logic validation to the server?

### ISSUE THREAD

**Issue by Utwo** [convergencelabs/convergence-project#176]
It's not necessary a feature request but more of a question. 
Is there a way I can add business logic validation to the server? How can I validate that data send from the client it's "valid" data?

**Comment by mmacfadden** [comment]
Greetings. Can you give an example of the kind of business logic you would like to validate?

**Comment by Utwo** [comment]
Yes, for example on a simple todo app. I want to limit the amount of todos a user can add. I'm also curious if I can define a validation schema that is checking data on backend side as well.

**Comment by mmacfadden** [comment]
At this time, this is not supported but it is on the road map.

**Comment by Utwo** [comment]
Ok, thank you!
